### PR TITLE
split voltron proxy into modules

### PR DESCRIPTION
**Pull request by mathieucarbou**
Fix required for #169

### CODE REVIEW

File: voltron-proxy/voltron-proxy-client/pom.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>voltron-proxy</artifactId>
+    <groupId>org.terracotta.voltron.proxy</groupId>
+    <version>5.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>voltron-proxy-client</artifactId>
+
```
Comment (anthonydahanne) [inline voltron-proxy/voltron-proxy-client/pom.xml]: isn't it required to set the maven compiler plugin and define the java.version to 1.6 ?
same thing for common

Comment (mathieucarbou) [inline voltron-proxy/voltron-proxy-client/pom.xml]: its defined as 1.6 by default at upper level ;-)

Comment (anthonydahanne) [inline voltron-proxy/voltron-proxy-client/pom.xml]: ok, thanks !

Comment (mathieucarbou) [inline voltron-proxy/voltron-proxy-client/pom.xml]: at least... it is... now :P

**Comment by anthonydahanne** [comment]
LGTM

**Pull request merged**